src/com/intellij/rt/coverage/data/JumpsAndSwitches.java

```diff
@@ -69,6 +69,12 @@ public SwitchData addSwitch(final int switchNumber, final int[] keys) {
     return mySwitches.get(switchNumber);
   }
 
+  public void removeSwitch(int switchNumber) {
+    if (mySwitches != null && 0 <= switchNumber && switchNumber < mySwitches.size()) {
```

Comment:
Is it really possible that mySwitches could be null here? Or switchNumber is out of bounds? Maybe it should not silently skip in such situations?
Same for removeJump